doc/src/modules/vector/api/classes.rst

```diff
@@ -2,13 +2,13 @@
 Essential Classes in sympy.vector (doctrings)
 =============================================
 
-CoordSysCartesian
-=================
+CoordSystem3D
```

Comment:
It depends on how much we would need to pack into a single class. If theres too much of case-based(Spherical/Euclidean/Cylindrical) processing to be done, the methods in a single class could bloat up pretty easily, making it difficult to follow. Plus, having too many parameters to a single initializer isn't better as compared to having different properly-labeled classes. Ofcourse this is looking into the future, just my 2 cents.